packages/web-runtime/src/composables/upload/useUpload.ts

```diff
@@ -181,7 +181,9 @@ const createDirectoryTree = ({
             unref(publicLinkPassword)
           )
         } else {
-          await client.files.createFolder(`${file.meta.webDavBasePath}/${folderToCreate}`)
+          try {
+            await client.files.createFolder(`${file.meta.webDavBasePath}/${folderToCreate}`)
+          } catch (ex) {}
```

Comment:
Additionally, we either name the error variable `error` or `e`, we might want to proceed with this naming scheme. 
Usually, we also log the error with `console.error`, if its worth ignoring the error completely we should write a comment, about why we do this.

Reply:
I used it because it throws an error if the folder already exists, i'll try to avoid it by checking beforehand because this way every error gets ignored